CONTRIBUTING.md

```diff
@@ -50,99 +52,102 @@ This section describes in some detail how changes can be made and proposed with
      ```
      git push
      ```
 
-   - To change existing commits you will have to [rewrite Git history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
-     Useful Git commands that can help a lot with this are `git commit --patch --amend` and `git rebase --interactive`.
-     With a rewritten history you need to force-push the commits using
+   - To change existing commits, [rewrite the Git history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
+     Useful Git commands for this are `git commit --patch --amend` and `git rebase --interactive`.
+     With a rewritten history you need to force-push the commits:
      ```
      git push --force-with-lease
      ```
 
-   - In case of merge conflicts you will also have to [rebase the branch](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on top of current `master`.
-     Sometimes this can be done [on GitHub directly](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch#updating-your-pull-request-branch), but if not you will have to rebase locally using
+   - In case of merge conflicts, you will have to [rebase the branch](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on top of the current **base branch**.
+     Sometimes this can be done [on GitHub directly](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch#updating-your-pull-request-branch).
+     To rebase locally:
      ```
      git fetch upstream
      git rebase upstream/master
      git push --force-with-lease
      ```
 
-   - If you need to change the base branch of the pull request, you can do so by [rebasing][rebase].
+     Use the base branch from step 1 instead of `upstream/master`.
+
+   - If you need to change the base branch, [rebase][rebase].
 
-8. If your pull request is merged and [acceptable for releases][release-acceptable] you may [backport][pr-backport] the pull request.
+8. If your PR is merged and [acceptable for releases][release-acceptable], you may [backport][pr-backport] it.
 
 ### Pull request template
 [pr-template]: #pull-request-template
 
-The pull request template helps determine what steps have been made for a contribution so far, and will help guide maintainers on the status of a change.
-The motivation section of the PR should include any extra details the title does not address and link any existing issues related to the pull request.
+The pull request template helps to determine which steps have been taken so far.
+The motivation section should include details the title does not address and link existing related issues.
 
-When a PR is created, it will be pre-populated with some checkboxes detailed below:
+When a PR is created, it will be pre-populated with these checkboxes:
```

Comment:
```suggestion
When a PR is created, it will be pre-populated with checkboxes discussed:
```

This phrasing enables some wiggle room should the pr template and contributing guide drift